mesonbuild/backend/vs2010backend.py

```diff
@@ -692,9 +692,12 @@ def create_basic_project(self, target_name, *,
             if target_ext:
                 ET.SubElement(direlem, 'TargetExt').text = target_ext
 
-            ET.SubElement(direlem, 'EmbedManifest').text = 'false'
-            if not gen_manifest:
-                ET.SubElement(direlem, 'GenerateManifest').text = 'false'
+            if gen_manifest == 'embed':
+                ET.SubElement(direlem, 'EmbedManifest').text = 'true'
+            else:
+                ET.SubElement(direlem, 'EmbedManifest').text = 'false'
+                if not gen_manifest:
+                    ET.SubElement(direlem, 'GenerateManifest').text = 'false'
```

Comment:
Something like this would be more symmetric, imho:

```suggestion
            ET.SubElement(direlem, 'EmbedManifest').text = 'true' if gen_manifest == 'embed' else 'false'
            ET.SubElement(direlem, 'GenerateManifest').text = 'true' if gen_manifest else 'false'
```

Reply:
Yes. I think GenerateManifest is true by default, but it shouldn't hurt to specify it explicitly. I'm not sure if it makes sense to have EmbedManifest if GenerateManifest is false. Another option would be to have `if ... elif ... else` and to only set what is needed. But personally, I like it as it is now.